### ISSUE TITLE
Can not connect any VM in the localhost mode

### ISSUE THREAD

**Issue by LITTLEFOLWOR** [ethereum/remix-project#4952]
I used to use localhost mode to study solidity.  But these days,  I found the connectable VM is less and less, a few days ago, I can not connect every VM any more! What's wrong with it?

**Comment by Aniket-Engg** [comment]
@LITTLEFOLWOR Can you delete `.states` folder from your workspace, reload Remix and check if accounts are loaded?

**Comment by LITTLEFOLWOR** [comment]
Yeah, it works, thank you for your advise. To solve this problem, I even installed ganache. But I have a question, what's the difference between official VM and local ganache?

**Comment by LITTLEFOLWOR** [comment]
@Aniket-Engg Yeah, it works, thank you for your advise. To solve this problem, I even installed ganache. But I have a question, what's the difference between official VM and local ganache?
